### PR TITLE
Document build steps using just esy

### PR DESCRIPTION
**Pull request by renatoalencar**
Sometimes using NixOS is not straightforward, so adding these docs in case a contributor needs to build and run the project using just esy.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -39,3 +39,14 @@ nix-build nix/ci/test.nix
 ```
 
 If that all passes, then congratulations! You are well on your way to becoming a contributor 🎉
+
+## Without NixOS
```
Comment (anmonteiro) [inline CONTRIBUTING.md:43]: I suggest adding a `Nix` section about, and name this section `Esy`

**Review by anmonteiro — COMMENTED** [review]
What about this: We extract just the installation section to one named "Nix", and add your instructions to a section named "Esy".

Then we can list the common parts below it, e.g. the `ocaml-tree` stuff, the commands that build the project, etc.

How does that sound?

**Comment by renatoalencar** [comment]
> What about this: We extract just the installation section to one named "Nix", and add your instructions to a section named "Esy".
> 
> Then we can list the common parts below it, e.g. the `ocaml-tree` stuff, the commands that build the project, etc.
> 
> How does that sound?

That's great. I'll do that.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,7 +6,12 @@ Thanks for your interest! Below we describe Melange's development setup for a fe
 
 - The [Nix](https://nixos.org/) package manager
 
-## Developing
+## Instalation
+
+You can both use [#nix](Nix) or [#esy](Esy) to setup your application, give preference to
+Nix since it can have more predictable environments, but you can run with Esy if your prefer.
```
Comment (anmonteiro) [inline CONTRIBUTING.md:12]:
```suggestion
Melange can be set up with [#nix](Nix) or [#esy](Esy). Instructions for both are detailed below.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -19,23 +24,34 @@ nix-shell --pure
 
 Once you're in the shell, you have access to `dune`, `node`, `yarn`, and all the other executables you need to get down to business.
 
+### Esy
+
+You can run without using Nix, you just need [esy](https://esy.sh/). Just straightforward, run `esy` and you are set up.
+You still need to make sure you runned `npm install` on the `ocaml-tree`directory.
```
Comment (anmonteiro) [inline CONTRIBUTING.md:30]:
```suggestion
In order to develop using [Esy](https://esy.sh/), you need to install it first. The usual way for doing that is to run `npm install -g esy`. Then run:

1. `esy install` to get all the project's dependencies
2. `esy build` to build the project.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -19,23 +24,34 @@ nix-shell --pure
...
+## Developing
+
 Before you try building with `dune`, be sure to install the local `ocaml-tree` project:
 
 ```sh
 cd ocaml-tree && npm install
 ```
 
-Here are some common commands you may find useful:
+And, here are some common commands you may find useful:
```
Comment (anmonteiro) [inline CONTRIBUTING.md:40]:
```suggestion
Here are some common commands you may find useful:
```

**Comment by renatoalencar** [comment]
Awesome suggestions, I committed them.

**Comment by anmonteiro** [comment]
Thanks!

**Pull request merged**